playbook/app/pb_kits/playbook/pb_selectable_card/docs/_selectable_card_single_select.jsx

```diff
@@ -23,7 +23,7 @@ class SelectableCardSingleSelect extends React.Component {
             name="gender"
             onChange={this.handleSelect.bind(this)}
             value="male"
-            {...props}
+            {...this.props}
```

Comment:
What's going on here with the swap from `{...props}` to `{...this.props}`?

Reply:
Is this the "double padding/margin" fix?

Reply:
This is an old school class based React component (instead of new functional comp) so I think you've gotta have `this.props` as opposed to just `props`.  Recent PR went through and added prop passes to all the React examples.  They likely misapplied props here without the `this.`.

Reply:
props aren't passed to render function on a class component, so they weren't doing anything as `...props`
Pairing with Jasper on this now

Reply:
Ok, my mistake, I didn't even notice the class structure. So many of our components are functional it just caught my eye. `this.props` is totally cool if we stick with that class component, but if it's possible to use hooks that would be cool too.

Reply:
Changed this over to functional, matching the rest of the react examples